plugins/aws/ec2/openCustomPorts.js

```diff
@@ -0,0 +1,95 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Open Custom Ports',
+    category: 'EC2',
+    description: 'Ensures that the defined ports are not exposed publicly',
+    more_info: 'Security groups should be used to restrict access to ports from known networks.',
+    link: 'https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html',
+    recommended_action: 'Modify the security group to ensure the ports are not exposed publicly.',
+    apis: ['EC2:describeSecurityGroups', 'STS:getCallerIdentity'],
+    settings: {
+        open_port_allowed_list: {
+            name: 'EC2 Allowed Open Ports',
+            description: 'A comma-delimited list of ports that indicates open ports allowed for any connection',
+            regex: '[a-zA-Z0-9,]',
+            default: [80, 443]
```

Comment:
this should be a string not an array. Example here: https://github.com/aquasecurity/cloudsploit/blob/master/plugins/aws/cloudformation/plainTextParameters.js#L17

Reply:
Like this: '80,443'?

Reply:
Yes but you will have to refactor the code to split the string into an array.